code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -424,9 +424,13 @@ GLOBAL_VAR_INIT(farm_animals, FALSE)
 		if(held_item)
 			if((butcher_results || guaranteed_butcher_results) && held_item.get_sharpness() && held_item.wlength == WLENGTH_SHORT)
 				var/used_time = 210
+				if(src.buckled && istype(src.buckled, /obj/structure/meathook))
+					used_time -= 30
+					visible_message("[user] begins to efficiently butcher [src] thanks to the meathook.")
```

Comment:
'thanks to the meathook' stuff will be weird after first time.

Reply:
noted